app/Http/Controllers/Slack/Registration.php

```diff
@@ -19,7 +19,10 @@ class Registration extends \App\Http\Controllers\BaseController
      */
     public function getNew()
     {
-        $this->authorize('register-slack');
+        if (!$this->account->can('register-slack')) {
```

Comment:
The `register-slack` Gate has other reasons it could error (e.g. if you're an international member) so this error message could be mis-leading